node/impl/full/gas.go

```diff
@@ -279,7 +279,7 @@ func gasEstimateCallWithGas(
 	// Try calling until we find a height with no migration.
 	var res *api.InvocResult
 	for {
-		res, err = smgr.CallWithGas(ctx, &msg, priorMsgs, ts)
+		res, err = smgr.CallWithGas(ctx, &msg, priorMsgs, ts, false)
```

Comment:
Doing this by default is non-ideal, and likely to break things in some cases - e.g. I'm not confident that lotus-miner will not have edge-cases related to this.

Can we either:
* Make this true by default, and possible to set with an env var
* Create a separate GasEstimateGasFast method, call that from GasEstimateMessageGas in lotus-gateway

Reply:
envvar good for today